docs/submitting-molecular-data.md

```diff
@@ -2,11 +2,86 @@
 id: submitting-molecular-data
 title: Submitting Molecular Data
 ---
+Molecular data will be submitted to your local Regional Data Processing Centre (RDPC). RDPCs are responsible for processing your programs molecular data according to the [Harmonization Pipeline](). If you are unsure what the correct RDPC is to submit to, please [contact the DCC](https://platform-ui.qa.argo.cancercollaboratory.org/contact).
```

Comment:
need an apostrophe "your program's molecular data"

"If you are unsure what the correct RDPC is to submit to, please.." change to
"If you are unsure which RDPC you should submit to, please contact.."